development/openstack-base-trusty-mitaka/README.md

```diff
@@ -57,42 +57,42 @@ All commands are executed from within the expanded bundle.
 
 In order to configure and use your cloud, you'll need to install the appropriate client tools:
 
-    sudo apt-get -y install python-novaclient python-keystoneclient \
-        python-glanceclient python-neutronclient
+    sudo add-apt-repository cloud-archive:ocata -y
+    sudo apt update
+    sudo apt install python-novaclient python-keystoneclient python-glanceclient \
+        python-neutronclient python-openstackclient -y
 
 ### Accessing the cloud
 
 Check that you can access your cloud from the command line:
 
     source novarc
-    keystone catalog
+    openstack catalog list
 
 You should get a full listing of all services registered in the cloud which should include identity, compute, image and network.
 
 ### Configuring an image
 
 In order to run instances on your cloud, you'll need to upload an image to boot instances from:
 
-    mkdir -p ~/images
-    wget -O ~/images/trusty-server-cloudimg-amd64-disk1.img \
-        http://cloud-images.ubuntu.com/trusty/current/trusty-server-cloudimg-amd64-disk1.img
-    glance image-create --name="trusty" --visibility public --progress \
-        --container-format=bare --disk-format=qcow2 \
-        < ~/images/trusty-server-cloudimg-amd64-disk1.img
+    curl http://cloud-images.ubuntu.com/xenial/current/xenial-server-cloudimg-amd64-disk1.img | \
```

Comment:
is it worth providing links to other images for other arches here?

Reply:
Yes, in a separate review IMO.